lntest/harness.go

```diff
@@ -772,7 +772,13 @@ func (n *NetworkHarness) RestartNode(node *HarnessNode, callback func() error,
 		unlockReq.RecoveryWindow = 1000
 	}
 
-	return node.Unlock(context.Background(), unlockReq)
+	if err := node.Unlock(context.Background(), unlockReq); err != nil {
+		return err
+	}
+
+	ctxc, done := context.WithTimeout(context.Background(), DefaultTimeout)
+	defer done()
+	return node.WaitForBlockchainSync(ctxc)
```

Comment:
Ah and this'll ensure that it detects w/e UTXOs were sent to it while it was offline in time?

Reply:
Yeah, and hopefully make the Neutrino tests a bit more stable in general by giving it time to fully sync to the chain after a restart.